### PR TITLE
Add Family Planning row to UNFPA report

### PR DESCRIPTION
**Pull request by Alastair-L**
### Issue #:
https://github.com/beyondessential/tupaia-backlog/issues/1156

### CODE REVIEW

File: packages/database/src/migrations/20200831024802-UpdateUNFPAReportToAddFPData-modifies-data.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+    name: '# women',
+    type: '$period',
+    periodType: 'quarter',
+    aggregationType: 'FINAL_EACH_QUARTER',
+    fillEmptyPeriods: true,
+  },
+  baselineColumns: [
+    {
+      name: 'Does this facility offer the service?',
+      dataElements: ['RHS3UNFPA536', 'RHS3UNFPA4121', 'RHS3UNFPA464', 'RHS3UNFPA536'],
```
Comment (rohan-bes) [inline packages/database/src/migrations/20200831024802-UpdateUNFPAReportToAddFPData-modifies-data.js:131]: The code `RHS3UNFPA536` is listed twice? Should there be a different baseline for the `Family Planning` data?

**Review by rohan-bes — COMMENTED** [review]
Just one question about the data for the baseline column for Family Planning. Honestly not sure, just double checking on that one

Happy to approve is that's the correct config 👍

File: packages/database/src/migrations/20200831024802-UpdateUNFPAReportToAddFPData-modifies-data.js (hunk 1 above)
Comment (Alastair-L) [reply packages/database/src/migrations/20200831024802-UpdateUNFPAReportToAddFPData-modifies-data.js:131]: Good catch! I didn't look at that, I'll double check that now

Comment (Alastair-L) [reply packages/database/src/migrations/20200831024802-UpdateUNFPAReportToAddFPData-modifies-data.js]: Wow, very good catch. That could have been bad...

**Review by rohan-bes — APPROVED** [review]
Awesome 😄 nice work!

**Pull request merged**